### ISSUE TITLE
App not starting until installing razzle globally

### ISSUE THREAD

**Issue by CPoncet** [jaredpalmer/razzle#1439]
### Current Behavior

Following the docs, I did `npx create-razzle-app test`, `cd test` and `npm start`, app wouldn't start and I was getting a `Error: cannot find module 'webpack'`.

I had to install razzle globally via `npm i -g razzle` to use `razzle start` and now the app is starting.

I couldn't find a similar issue, maybe it is related to my setup?

### Expected behavior

If the docs are correct the app should start without having to install razzle globally and without using `razzle start` command

### Reproducible example

Plain simple `npx create-razzle-app`

### Your environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Razzle           | 4.0.0-canary.1
| Razzle Plugins   | none
| Node             | 14.13.0
| Browser          | Chrome 85.0.4183.121
| npm/Yarn         | npm 6.14.8
| Operating System | MacOS 10.15.7
| TypeScript       | none
| React            | 16.13.1

**Comment by fivethreeo** [comment]
Webpack is a peerdependency in 4.0.0 canary so you need to install it.

But create-razzle-app should install it, need to look into it :)

**Comment by fivethreeo** [comment]
I usually use yarn.

**Comment by CPoncet** [comment]
> Webpack is a peerdependency in 4.0.0 canary so you need to install it.
> 
> But create-razzle-app should install it, need to look into it :)

I tried installing webpack manually but it didn't work either.

**Comment by fivethreeo** [comment]
Try npm start and not razzle start.

**Comment by fivethreeo** [comment]
I did

```bash
npx create-razzle-app@canary test
cd test
npm start
```

**Comment by fivethreeo** [comment]
You could try

```bash
export NODE_ENV=development
npx create-razzle-app@canary test
cd test
npm start
```

**Comment by CPoncet** [comment]
> I did
> 
> ```shell
> npx create-razzle-app@canary test
> cd test
> npm start
> ```

Ok I can confirm it works with `create-razzle-app@canary` but not with `create-razzle-app`

**Comment by fivethreeo** [comment]
Fixed in v3.3.5
